packages/web3-types/src/eth_types.ts

```diff
@@ -83,6 +83,7 @@ export type TransactionOutput = {
 	readonly from?: HexString;
 	readonly input: string;
 	readonly gas?: Numbers;
+	readonly gasPrice?: Numbers;
```

Comment:
This change will not fix issue, this type is used in formatter functions and those formatters are only for backward compatibility.